json_serializable/test/default_value/default_value.dart

```diff
@@ -6,12 +6,20 @@
 
 import 'package:json_annotation/json_annotation.dart';
 
+import 'default_value_interface.dart' as dvi hide Greek;
+import 'default_value_interface.dart' show Greek;
+
 part 'default_value.g.dart';
 
 const _intValue = 42;
 
+dvi.DefaultValue fromJson(Map<String, dynamic> json) =>
```

Comment:
nit: move to factory (just for best practices)? Generally you don't want to expose a top level method with such a generic name.

Reply:
I do it so this code is easier – I want something I can tear off and I can't tear off a ctor

https://github.com/dart-lang/json_serializable/blob/03fa1818eca2fb975a31b1602f86da01140d5299/json_serializable/test/default_value/default_value_test.dart#L43-L44